### PR TITLE
move rubygem-jquery-ui-rails package to katello

### PR DESCRIPTION
**Pull request by MariaAga**
for https://github.com/Katello/katello/pull/10982 and https://github.com/theforeman/foreman/pull/10142
I ran:
`./remove_package rubygem-jquery-ui-rails` 
`./add_gem_package.sh jquery-ui-rails foreman_plugin katello`

### CODE REVIEW

**Review by ekohl — CHANGES_REQUESTED** [review]
Technically I'd prefer to use `git mv` on the package directory **if** we're moving it. Then it also belongs in the `katello` comps, not `foreman-plugins`.

But I said **if**. I'd be OK with keeping this in Foreman's repo for now. Moving it creates some churn and it's really just a build-time dependency that we want to get rid of anyway. Obviously, this wouldn't be the first time we say that and then take forever to execute on it. Still, I think it's acceptable.

I think what @m-bucher meant is that currently the dependency is listed in `foreman.spec` but should be in `rubygem-katello.spec`.

**Comment by MariaAga** [comment]
Thanks @ekohl, is this the correct way to do it?

**Review by ekohl — CHANGES_REQUESTED** [review]
I always use https://github.com/ekohl/theforeman-plumbing/blob/master/fm-packaging-update-deps to update the dependencies. Once we drop EL8 in Foreman 3.13 then we can use https://github.com/theforeman/foreman-packaging/pull/8414 to automatically generate those sections based on the sources.

I wouldn't mind it if we merged the PRs and I can submit the update PRs for you. Ideally this whole complex area is going away and there's little point in learning it now.

File: packages/foreman/foreman/foreman.spec [hunk 1]
```diff
@@ -198,7 +198,6 @@ BuildRequires: (npm(react-intl) >= 2.8.0 with npm(react-intl) < 3.0.0)
 # end package.json dependencies BuildRequires
 
 # start specfile assets BuildRequires
-BuildRequires: (rubygem(jquery-ui-rails) >= 6.0 with rubygem(jquery-ui-rails) < 7.0)
```
[the remark this answers is not in the document]
Comment (MariaAga) [reply packages/foreman/foreman/foreman.spec:201]: I made https://github.com/theforeman/foreman/pull/10142 to remove it

File: packages/katello/rubygem-katello/rubygem-katello.spec [hunk 2]
```diff
@@ -76,6 +76,10 @@ BuildRequires: (npm(react-bootstrap) >= 0.32.1 with npm(react-bootstrap) < 1.0.0
 BuildRequires: (npm(use-deep-compare-effect) >= 1.6.1 with npm(use-deep-compare-effect) < 2.0.0)
 # end package.json dependencies BuildRequires
 
+# start specfile assets BuildRequires
+BuildRequires: (rubygem(jquery-ui-rails) >= 6.0 with rubygem(jquery-ui-rails) < 7.0)
+# end specfile assets BuildRequires
```
[the remark this answers is not in the document]
Comment (MariaAga) [reply packages/katello/rubygem-katello/rubygem-katello.spec:81]: Not sure if I should keep both BuildRequires and Requires then?

[the remark this answers is not in the document]
Comment (ekohl) [reply packages/katello/rubygem-katello/rubygem-katello.spec:81]: For `rubygem-katello.spec` (and all `rubygem-*.spec` really) it should autogenerate `Requires`, so no need for that.

[the remark this answers is not in the document]
Comment (MariaAga) [reply packages/katello/rubygem-katello/rubygem-katello.spec:81]: removed the `Requires`

**Comment by MariaAga** [comment]
@ekohl Thanks for the review, how can I progress this?

**Comment by adamruzicka** [comment]
[test rpm-copr]

**Comment by MariaAga** [comment]
split into https://github.com/theforeman/foreman-packaging/pull/11416 https://github.com/theforeman/foreman-packaging/pull/11417

**Pull request closed without merging**